src/pages/workspace/WorkspacePageWithSections.tsx

```diff
@@ -41,9 +40,6 @@ type WorkspacePageWithSectionsProps = WithPolicyAndFullscreenLoadingProps &
         /** The text to display in the header */
         headerText: string;
 
-        /** The route object passed to this page from the navigator */
-        route: PolicyRoute;
```

Comment:
why it was removed?

Reply:
@kubabutkiewicz Because it's inherited from `WithPolicyAndFullscreenLoadingProps`